### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by saintsebastian**
This is a starting point. Of course, we would need to have all copy translated to French (or write it in French and translate to English). Feel free to share it with any other WIT member or ally who can improve my clanky start

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
 # OPE
-One Page for Equity
+
+![OPE logo](logo-ope-reserve.png)
+
+## One Page for Equity
+
+This projects aims to create a dashboard with variety of gender based information and visualisation.
+
+According to law french companies now are required to measure and publish information on following aspects"
```
Comment (SdgJlbl) [inline README.md:9]: French
s/-"/+:

Comment (sophisur) [reply README.md:9]: maybe a comma between law and french

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+ - pay comparison for men, women and non-binary people based on their age and job position
+ - gap in the rate of individual salary increases between women and men
+ - comparison in promotions between women and men and representation of different genders in management
+ - effect of parenthood and maternity leave on salaries
+ - representation of genders in companies top 10 highest-paid employees
```
Comment (sophisur) [inline README.md:14]: Could be nice to cap the first letter and a dot at the end ?

**Comment by saintsebastian** [comment]
@SarahDiotGirard @sophisur I've updated this PR with your comments in mind

**Pull request merged**